TACC/Core-Portal
# Bug/WP-267: JS Warnings: Adjust button usage and pass in required fields

## Overview

These are fixes for warnings in code and tests

* Button component implementation looks for `children` property to be a string. Fixing bad usages.
* Required fields are not passed from tests.

## Related

* [WP-260](https://tacc-main.atlassian.net/browse/WP-260)  - Button component children prop validation failure
* [WP-267](https://tacc-main.atlassian.net/browse/WP-267) - Disconnect and Connect are required fields

## Changes
* Make the button component `children` to be a single node or string.
* Pass disconnect and connect attributes to integration component in Manage Account. It is required field and value is not provided.


## Testing

1. Run tests in client folder 
npm run test -- src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectFileListing.test.js
npm run test -- src/components/ManageAccount/tests/ManageAccountTables.test.js 

2. Go Manage Account in cep.test of your user and google drive disconnect or connect (only one) of them is shown.

3. Go to a shared workspace and ensure Manage Team/View Team link button is shown.


## UI



## Notes
